tcms/core/tests/test_forms.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+
+from tcms.core.forms.fields import UserField
+from tcms.tests.factories import UserFactory
+
+
+class TestUserField(TestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = UserFactory(username='admin', email='admin@example.com')
+        cls.user_field = UserField()
+
+    @classmethod
```

Comment:
This isn't a class method. Signature is `def setUp(self):`, see LoggedInTestCase and https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

My guess is that your current definition overrides what unittest expects by default and that's why it doesn't break.